### PR TITLE
Class mappings docs

### PR DESCRIPTION
**Pull request by adambrinkman**

### CODE REVIEW

File: docs-src/docs/dataset-exploration/uploading-predictions.md [hunk 1]
```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
     print(f"Request error: {e}. {msg}")
 ```
 
+Predictions groups connected to segmentation datasets require one extra parameter called `classMapping`. This is used to map the class names 
+in the predictions to the class names in the annotations. The `classMapping` parameter is a list of dictionaries, where each dictionary
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-exploration/uploading-predictions.md:62]: I think we could be even more clear that this affects the matching and what will be marked as disagreement in the gallery.

File: docs-src/docs/dataset-exploration/uploading-predictions.md [hunk 2]
```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
...
+contains the keys `annotated` and `predicted`. The `annotated` key is the class name in the annotations, and the `predicted` key is the 
+class name in the predictions. All class names in the predictions and the annotations must be present in the class mappings, even if they 
+don't need to be mapped.
+
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-exploration/uploading-predictions.md:66]: Maybe we could expand the example a little by also adding
`{"annotated": "oak", "predicted": "tree"},`
and a short description like "if you have annotated different species of trees, but predicts only whther it is a tree or not".

File: docs-src/docs/dataset-exploration/uploading-predictions.md [hunk 3]
```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
...
+```python
+example_body = {
+    "name": "My predictions group",
+    "description": "A description of my new predictions group",
+    "classMapping": [
+        {"annotated": "oak", "predicted": "tree"},
+        {"annotated": "only_in_annotations"}
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-exploration/uploading-predictions.md:73]: Also add something like
`{"annotated": "_background", "predicted": "not_tree"}`

File: docs-src/docs/dataset-exploration/uploading-predictions.md [hunk 4]
```diff
@@ -56,6 +58,23 @@ except requests.exceptions.RequestException as e:
     print(f"Request error: {e}. {msg}")
 ```
 
+Predictions groups connected to segmentation datasets require one extra parameter called `classMapping`. This is used to map the class names 
+in the predictions to the class names in the annotations. The `classMapping` parameter is a list of dictionaries, where each dictionary
+contains the keys `annotated` and `predicted`. The `annotated` key is the class name in the annotations, and the `predicted` key is the 
+class name in the predictions. All class names in the predictions and the annotations must be present in the class mappings, even if they 
+don't need to be mapped.
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-exploration/uploading-predictions.md:65]: For the annotations are non-segmented areas given the class name `_background`.

**Review by andreas-lindholm — APPROVED** [review]
Added some suggestions for clarifications

File: docs-src/docs/dataset-exploration/uploading-predictions.md (hunk 1 above)
Comment (adambrinkman) [reply docs-src/docs/dataset-exploration/uploading-predictions.md:62]: Good point, will fix

File: docs-src/docs/dataset-exploration/uploading-predictions.md (hunk 3 above)
Comment (adambrinkman) [reply docs-src/docs/dataset-exploration/uploading-predictions.md:73]: Why should there be an underscore before background?

**Pull request merged**